app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
```

Comment:
"We suggest you..." - note that this error suggests this document has not been run through a spelling and grammar checker like Grammarly - please do so.

Reply:
@coopr Not sure about Grammarly, but Oxford Dictionary and McMillan Dictionary don't expect a pronoun for the verb "suggest", https://en.oxforddictionaries.com/definition/suggest, https://www.macmillandictionary.com/dictionary/british/suggest.
However they say to change the infinitive to a gerund which will become "We suggest following the below steps..."